3_javascript/javascript3/week1/Giphy/scripts.js

```diff
@@ -0,0 +1,53 @@
+domListener();
+function domListener() {
+  document.addEventListener("DOMContentLoaded", domLoader);
+}
+
+function domLoader() {
+  const inputSearch = document.getElementById("search");
+  const clickSearch = document.getElementById("click");
+  const numberOfSearch = document.getElementById("quantity");
+  const resultOfSearch = document.getElementById("result");
+
+  let toSearch = "";
+  let limitOfSearch = 25;
+
+  inputSearch.addEventListener(
```

Comment:
If I am not mistaken, this will run on every key entered, which there is no need to do. 
You need to take and lowercase and use the value only when a search is executed, so you could save resources by only doing this in the search execution function 💡

Reply:
Thanks Karo! Super valuable review for me. I applied the changes :)